src/Core/src/Platform/Android/ImageViewExtensions.cs

```diff
@@ -29,7 +30,20 @@ public static void UpdateAspect(this ImageView imageView, IImage image)
 
 		public static void UpdateIsAnimationPlaying(this Drawable? drawable, IImageSourcePart image)
```

Comment:
I had to fix something similar earlier this year: https://github.com/xamarin/xamarin-android/pull/8594 I think you can fix this with a few `[DynamicDependency]` attributes to this method:
```c#
[DynamicDependency(nameof(IAnimatable))]
[DynamicDependency(nameof(AnimationD)rawable)]
[DynamicDependency(nameof(GifDrawable))]
[DynamicDependency(nameof(AnimatedImageDrawable))]
```

Reply:
Does no C# code use any of the above types? What @simonrozsival suggests seems like the right idea.

Something similar also happens here:
* https://github.com/xamarin/xamarin-android/blob/b521487d0226db265baaa83f0f53e2e75497c420/tests/Mono.Android-Tests/Android.Graphics/NinePatchTests.cs#L27-L34

Since no C# code uses `NinePatchDrawable` it's linked away. `Mono.Android.dll` and all the AndroidX libraries mark themselves trimmable.

Reply:
The issue is that I don't know what types are being used - I know of _some_ but it could be _any_ type of drawable. I would have to list all of the known ones and then it would still not be super perfect because of third parties.

Also, since we don't use the type in C# - all the loading code is in Java now - we are otentially forcing a whole bunch of C# types to exist when we don't really need it.

Reply:
It doesn't seem like `[DynamicDependency]` will be applicable in this case then. Doing the JavaCast might be the best way to do that in this case. Hopefully Proguard won't be messing up the Java side of things :)